libmate-desktop/display-name.c

```diff
@@ -22,7 +22,7 @@
 
 /* Author: Soren Sandmann <sandmann@redhat.com> */
 
-#include <config.h>
+#include "config.h"
```

Comment:
Then in sub directory, you can do it like this:
```
shared_library(
  ...
  include_directories: top_srcdir,
  ...
)

Reply:
Aware of said technique, but this will avoid nearly impossible to diagnose build errors if
some package mistakenly installs `config.h` (either due to the distro packager themself
or the upstream project) into the system.
https://github.com/pepe2k/u-boot_mod/issues/148